include/mbgl/ios/MGLMapView.h

```diff
@@ -404,6 +404,30 @@ IB_DESIGNABLE
 - (void)setZoomLevel:(double)zoomLevel animated:(BOOL)animated;
 
 /**
+ * The minimum zoom level the can be shown at.
```

Comment:
It’s worth noting here that the minimum zoom level can still be [constrained](https://github.com/mapbox/mapbox-gl-native/blob/8357b3716bfc563c2fb436a51a42dc6b6bfe3c32/src/mbgl/map/transform_state.cpp#L365-L368) regardless of this property to prevent showing multiple copies of the world at the same time. This is relevant whenever the view is much wider than it is tall.